### PR TITLE
New User deploy permissions

### PR DESCRIPTION
**Pull request by jonat123**
New article describing the new permissions for users and deploy

### CODE REVIEW

File: Umbraco-Cloud/Deployment/Users-On-Cloud/index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+### Set up Permissions for transfers and restores
+
+There is two ways that you can set up these permissions.
+You can either create a new User Group and set the permissions there or you can edit an existing group.
+
+To create or edit a user group, go to the user section of the backoffice.
+
+In the right corner, you need to click on "Groups".
+From here you are able to either create a new or edit an existing User Group.
```
Comment (Jay-umbr) [inline Umbraco-Cloud/Deployment/Users-On-Cloud/index.md:29]:
```suggestion
From here you are able to either create a new User Group, or edit an existing one.
```

File: Umbraco-Cloud/Deployment/Users-On-Cloud/index.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+![User Groups](images/Users.png)
+
+Click on "Create group" and scroll down and look under "Default permissions" where you can see the three options:
+
+![User Groups](images/default_permisions.png)
+
+You can decide whether the users in the new User Group can restore, restore partial or transfer content.
```
Comment (Jay-umbr) [inline Umbraco-Cloud/Deployment/Users-On-Cloud/index.md:37]:
```suggestion
You can decide whether the users in the new User Group can restore, partially restore, or transfer content.
```

File: Umbraco-Cloud/Deployment/Users-On-Cloud/index.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+To edit an already existing user group go to the User group you want to edit, e.g Editors or Writers and set the permissions from there.
```
Comment (Jay-umbr) [inline Umbraco-Cloud/Deployment/Users-On-Cloud/index.md:39]:
```suggestion
To edit an already existing user group, go to the User group you want to edit, e.g Editors or Writers, and set the permissions from there.
```

File: Umbraco-Cloud/Deployment/Users-On-Cloud/index.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+As mentioned it is also possible to set Granular permissions for a specific content node on your cloud project.
+
+You can set the permission when you are creating or editing an existing user group.
+
+In the bottom of the user group, you can add the setting for Granular permission for your content nodes.
+
+![Granular permission](images/Granular.png)
+
+When you click add, you can choose which content node on your project that you want to set the Granular settings for.
```
Comment (Jay-umbr) [inline Umbraco-Cloud/Deployment/Users-On-Cloud/index.md:51]:
```suggestion
When you click "add", you can choose which content node on your project that you want to set the Granular settings for.
```

File: Umbraco-Cloud/Deployment/Users-On-Cloud/index.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+![Granular content node](images/Granular_node.png)
+
+When you have choosen the node that you want to set the settings for, you can then set the permissions for restore, partial restore and queueing content for transfer.
```
Comment (Jay-umbr) [inline Umbraco-Cloud/Deployment/Users-On-Cloud/index.md:55]:
```suggestion
When you have chosen the node that you want to set the settings for, you can then set the permissions for restore, partial restore, and queueing content for transfer.
```

**Review by Jay-umbr — COMMENTED** [review]
Some grammar mistakes present. I suggested changes. Otherwise - a very good article

**Review by sofietoft — CHANGES_REQUESTED** [review]
Super article, @jonat123 !

Be careful of the grammar ;)
And also, make sure you're consistent with capitalizing names and specific Umbraco terms.
I've made some suggestions - let me know what you think!

File: Umbraco-Cloud/Deployment/Users-On-Cloud/index.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
+# Users on Umbraco Cloud
+
+On Umbraco Cloud, users work almost identical as they do on a normal installation of Umbraco, however, you have a few more settings available for the backoffice users on Umbraco Cloud.
```
Comment (Jay-umbr) [inline Umbraco-Cloud/Deployment/Users-On-Cloud/index.md:3]:
```suggestion
On Umbraco Cloud, users work in almost the same way as on a normal installation of Umbraco; however, there are a few more settings available for the backoffice users on Umbraco Cloud.
```

File: Umbraco-Cloud/Deployment/Users-On-Cloud/index.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+In this article, we will show how users work and the different settings on Umbraco Cloud.
```
Comment (Jay-umbr) [inline Umbraco-Cloud/Deployment/Users-On-Cloud/index.md:5]:
```suggestion
In this article, we will show how users work on Umbraco Cloud, as well as explain the different settings for users.
```

File: Umbraco-Cloud/Deployment/Users-On-Cloud/index.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+
+## Adding users on Umbraco Cloud
+
+There are two ways of adding a user to your backoffice on Umbraco Cloud.
+
+You can add them as a [Team member](/Umbraco-Cloud/Set-Up/Team-Members/) from the project portal on Umbraco Cloud.
+
+They will by default when added to the project as a team member be added to the backoffice as administrators.
```
Comment (Jay-umbr) [inline Umbraco-Cloud/Deployment/Users-On-Cloud/index.md:13]:
```suggestion
By default, when added to the project as a team member, they will be added to the backoffice as administrators.
```

File: Umbraco-Cloud/Deployment/Users-On-Cloud/index.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+
+users can also be invited directly from the backoffice of your Umbraco Cloud project where you can give them different user permissions.
```
Comment (Jay-umbr) [inline Umbraco-Cloud/Deployment/Users-On-Cloud/index.md:15]:
```suggestion
Users can also be invited directly from the backoffice of your Umbraco Cloud project, from where you can give them different permissions.
```

File: Umbraco-Cloud/Deployment/Users-On-Cloud/index.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+
+Check out the [Users article](/Getting-Started/Data/Users/) for an in-depth explanation about Umbraco users in general.
+
+:::note
+users are environment-specific on Umbraco Cloud, this means that they will not be transferred over when doing a deployment to the next environment, this means that they will need to be added on the different environments on Umbraco Cloud.
```
Comment (Jay-umbr) [inline Umbraco-Cloud/Deployment/Users-On-Cloud/index.md:20]:
```suggestion
Users are environment-specific on Umbraco Cloud. This means that they will not be transferred over when doing a deployment to the next environment - they will need to be added on the different environments on Umbraco Cloud.
```

File: Umbraco-Cloud/Deployment/Users-On-Cloud/index.md [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+:::
+
+## User group permissions for transfers and restores
+
+A feature on Umbraco Cloud is the possibility to control which users have access to transferring and restoring content and media on your Umbraco Cloud project.
```
Comment (Jay-umbr) [inline Umbraco-Cloud/Deployment/Users-On-Cloud/index.md:25]:
```suggestion
On Umbraco Cloud, it is possible to control which users have access to transferring and restoring content and media on your Umbraco Cloud project.
```

File: Umbraco-Cloud/Deployment/Users-On-Cloud/index.md [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+
+This can be done when creating a new User Group or when editing an existing group.
+
+You have the option to decide whether a specific User Group has permission to do a restore, partial restore or if they can queue content for transfer to the next environment.
```
Comment (Jay-umbr) [inline Umbraco-Cloud/Deployment/Users-On-Cloud/index.md:29]:
```suggestion
You have the option to decide whether a specific User Group has permission to do a restore, partial restore, or queue content for transfer to the next environment.
```

File: Umbraco-Cloud/Deployment/Users-On-Cloud/index.md [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+
+To edit an already existing User Group, go to the User Group you want to edit, e.g Editors or Writers, and set the permissions from there.
+
+### Granular Permissions
+
+As mentioned it is also possible to set Granular permissions for a specific content node on your cloud project.
+
+You can set the permission when you are creating or editing an existing user group.
+
+In the bottom of the User Group, you can add the setting for Granular permission for your content nodes.
```
Comment (Jay-umbr) [inline Umbraco-Cloud/Deployment/Users-On-Cloud/index.md:60]:
```suggestion
At the bottom of the User Group, you can add the setting for Granular permission for your content nodes.
```

File: Umbraco-Cloud/Deployment/Users-On-Cloud/index.md [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+
+![Granular permission](images/Granular.png)
+
+When you click "Add", you can choose which content node on your project that you want to set the Granular settings for.
```
Comment (Jay-umbr) [inline Umbraco-Cloud/Deployment/Users-On-Cloud/index.md:64]:
```suggestion
When you click "Add", you can choose the content node which you want to set the Granular settings for.
```

**Review by Jay-umbr — CHANGES_REQUESTED** [review]
Made suggestions. Some minor mistakes. Some stuff is phrased in a weird way

File: Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+
+You can add them as a [Team member](/Umbraco-Cloud/Set-Up/Team-Members/) from the project portal on Umbraco Cloud.
+
+By default, when added to the project as a team member, they will be added to the backoffice as administrators.
+
+Users can also be invited directly from the backoffice of your Umbraco Cloud project, from where you can give them different permissions.
+
+Check out the [Users article](/Getting-Started/Data/Users/) for an in-depth explanation about Umbraco users in general.
+
+:::note
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md:19]: Should we add to this note, that if they're added as Team Members then they will be added to ALL environments?

File: Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md [hunk 16]
```diff
@@ -0,0 +1,70 @@
...
+### Set up Permissions for transfers and restores
+
+There are two ways that you can set up these permissions:
+
+- Create a new User Group
+- Edit an existing one
+
+To create or edit a User Group, go to the User section of the backoffice.
+
+In the right corner, you need to click on "Groups", from here you are able to either create a new User Group or edit an existing one.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md:42]:
```suggestion
In the right corner, you need to click on "Groups", and from here you are able to either create a new User Group or edit an existing one.
```

File: Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md [hunk 17]
```diff
@@ -0,0 +1,70 @@
...
+
+You can decide whether the users in the new User Group can restore, partially restore, or transfer content.
+
+To edit an already existing User Group, go to the User Group you want to edit, e.g Editors or Writers, and set the permissions from there.
+
+### Granular Permissions
+
+As mentioned it is also possible to set Granular permissions for a specific content node on your cloud project.
+
+You can set the permission when you are creating or editing an existing user group.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md:58]:
```suggestion
You can set the permission when you are creating or editing an existing User Group.
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Great article @jonat123 !

I've added some more comments.
Also, how about adding the steps to set permissions (lines 40 to 48ish), into a list instead?
That would make it easier to follow the steps, I think 🤔 
What do you think?

**Comment by sofietoft** [comment]
Forgot to mention one thing:
When creating **new** articles on the docs - make sure you add YAML tags to the very beginning of the article.
See more here: https://our.umbraco.com/documentation/Contribute/File-Naming-Conventions/#adding-meta-data

File: Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md [hunk 18]
```diff
@@ -0,0 +1,76 @@
...
+
+    ![User Groups](images/default_permisions.png)
+
+3. From here you can decide whether the users in the new User Group can restore, partially restore, or transfer content.
+
+4. To edit an already existing User Group, go to the User Group you want to edit, e.g Editors or Writers, and set the permissions from there.
+
+### Granular Permissions
+
+As mentioned it is also possible to set Granular permissions for a specific content node on your cloud project.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md:62]:
```suggestion
As mentioned it is also possible to set Granular permissions for a specific content node on your Cloud project.
```

**Review by sofietoft — COMMENTED** [review]
Juuuust a few mionr things 😁

**Pull request merged**